venv/.gitignore

```diff
@@ -0,0 +1 @@
+.data.txt
```

Comment:
please move `.gitignore` to root project path (a place here you store `requirements.txt` for example)